src/main/java/eu/dissco/orchestration/backend/database/jooq/tables/MachineAnnotationService.java

```diff
@@ -150,6 +150,18 @@ public Class<MachineAnnotationServiceRecord> getRecordType() {
      */
     public final TableField<MachineAnnotationServiceRecord, JSONB> DATA = createField(DSL.name("data"), SQLDataType.JSONB.nullable(false), this, "");
 
+    /**
+     * The column <code>public.machine_annotation_service.environment</code>.
+     * Environmental variables to supply to the MAS, non-sensitive
+     */
+    public final TableField<MachineAnnotationServiceRecord, JSONB> ENVIRONMENT = createField(DSL.name("environment"), SQLDataType.JSONB, this, "Environmental variables to supply to the MAS, non-sensitive");
```

Comment:
Oeh do we need to store them separately and indexed? They will be in the "data" part anyway. I would say that is enough